### ISSUE TITLE
Increase the number of objects returned by Powershell GraphApi using the -top switch

### ISSUE THREAD

**Issue by prowland2020** [nicolonsky/ConditionalAccessDocumentation#1]
Thanks for your code. I ran into some issue with enumerating the Service Principals and had a number of AppID references, can I suggest the following changes to the code,

There is a hidden switch add a -top 999 to the Get-MgServicePrincipal call, as by default the Powershell GraphAPI limits the number of objects returned to 100.  

The -top 999 can be added to the Get-MgConditionalAccessPolicy, Get-MgConditionalAccessNamedLocation

**Comment by nicolonsky** [comment]
Thanks for your feedback. I am aware of the paging issues but refused to implement this "999" entries workaround because it does not make sense in my opinion (better add proper support for paging). So I will just wait until the Microsoft Graph modules implement / ship with proper paging support. It seems that we can expect such a feature shipping within the next module version: https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/263#issue-434731942

**Comment by prowland2020** [comment]
Hi Nicola

A new version of Powershell Microsoft.Graph Module has been released version 0.70. That resolves the limitation of only being able to return 999 objects from the Directory, release notes are here : https://github.com/microsoftgraph/msgraph-sdk-powershell/releases. I changed the -top 999 to a -All flag in the version of the code I was running and it will return all of the objects.

Regards
Paul
